mongodbatlas/resource_mongodbatlas_event_trigger_test.go

```diff
@@ -207,8 +208,8 @@ func TestAccConfigRSEventTriggerAuth_eventProcessor(t *testing.T) {
 	var (
 		resourceName = "mongodbatlas_event_trigger.test"
 		projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-		awsAccountID = os.Getenv("AWS_ACCOUNT_ID")
-		awsRegion    = os.Getenv("AWS_REGION")
+		awsAccountID = os.Getenv("AWS_EVENTBRIDGE_ACCOUNT_ID")
+		awsRegion    = strings.ReplaceAll(strings.ToLower(os.Getenv("AWS_REGION")), "_", "-")
```

Comment:
[nit] extract to a method, there is existing `valRegion()` in provider.go that does opposite of this except for `network_peering` resources

Reply:
Once I push https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1549 I can re-use that.